### DISCUSSION TITLE
List Notes with a Certain Variable Value/Two-Way Relations

### DISCUSSION THREAD

**Discussion by Bluemoondragon07** [SilentVoid13/Templater#1037] [Help] [answered]
Hi. I'm new to using the templater plug-in, and I was wondering if anyone knew how to get a list of files with a certain variable value. 

For example, have a note list all files where the variable `motherLocation` = "California". Really, I was wondering if two-way relation field pairs like Mother Location and Sub-location are possible.

Really, I want to have a **Location** note with two variables: `motherLocation` and `subLocations`. `motherLocation` gets its value from user input upon note creation--it will allow the user to select files from the **`/Locations/`** folder. `subLocations` will list all files in the **`/Locations/`** folder where `motherLocation` = `tp.file.title`. Therefore, creating a sort of two-way relation.

Does anyone know how to achieve this?

**Comment by Zachatoo** [comment, marked as answer]
It would be possible to apply once as a template, though it would not be able to detect if you added a value to one of your frontmatter fields later.

This seems like a better use case for the dataview plugin potentially?

**Reply by Bluemoondragon07** [reply]
Ah, you're right! Dataview! The solution was under my nose the whole time. I can just have one field, `motherLocation`, and in each Location note have a dataview query. Something like:
```dataview
LIST
WHERE motherLocation = this.file.name)
```

Thanks!
